lms/urls.py

```diff
@@ -985,7 +985,10 @@
         settings.PROFILE_IMAGE_BACKEND['options']['base_url'],
         document_root=settings.PROFILE_IMAGE_BACKEND['options']['location']
     )
+    # TODO: re-enable this after removing the URL below
+    # urlpatterns += url(r'^template/(?P<template>.+)$', 'openedx.core.djangoapps.debug.views.show_reference_template')
 
+# TODO: DO NOT MERGE
```

Comment:
I'll remove this before merging. I need it on to be able to see the templates on my sandbox.